client/src/pages/private/ParticipantTable.js

```diff
@@ -412,10 +418,13 @@ export default () => {
           ];
 
         if (reducerState.tabValue === 'Hired Candidates') {
```

Comment:
```suggestion
        if (reducerState.tabValue === 'Hired Candidates') {
        // remove engage, siteName, and status columns
```
Possibly add a comment here to explain the filter?